src/App.js

```diff
@@ -1,23 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
+import Navbar from './components/Navbar';
+import CountriesList from './components/CountriesList';
+import { Route, Routes } from 'react-router-dom';
+import CountryDetails from './components/CountryDetails';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
+  const [countries, setCountries] = useState([])
+
+  useEffect(() => {
+    axios.get('https://ih-countries-api.herokuapp.com/countries')
+    .then(result => setCountries(result.data))
```

Comment:
Remember to always use the `.catch()` in your promises

```suggestion
    .then(result => setCountries(result.data))
    .catch(error => /* something to handle the error */)
```